package.json

```diff
@@ -26,6 +26,7 @@
     "@vue/cli-service": "^3.3.0",
     "@vue/eslint-config-prettier": "^4.0.1",
     "lint-staged": "^8.1.0",
+    "material-design-icons-iconfont": "^4.0.4",
```

Comment:
This is not needed, we already have material design icons set up

Reply:
But it was not working for me

Reply:
Try pulling new changes from master and then adding these commits on top of it. I added those in one of your PR, that change is not in your branch